src/pages/identity/Identity.js

```diff
@@ -1,10 +1,29 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import { connectIdmWallet } from 'react-idm-wallet';
+import { ModalTrigger } from '@nomios/web-uikit';
+import EditProfile from '../../modals/edit-profile';
 
-const Identity = ({ match }) => <div>This is the identity page of { match.params.id }!</div>;
+const Identity = ({ match, isLoaded }) => {
+    if (!isLoaded) {
+        return null;
+    }
+
+    return (
+        <Fragment>
+            <div>This is the identity page of { match.params.id }!</div>
+            <ModalTrigger modal={ <EditProfile id={ match.params.id } /> }>
+                <button>Edit Profile</button>
+            </ModalTrigger>
+        </Fragment>
+    );
+};
 
 Identity.propTypes = {
     match: PropTypes.object.isRequired,
+    isLoaded: PropTypes.bool.isRequired,
 };
 
-export default Identity;
+export default connectIdmWallet((idmWallet) => () => ({
```

Comment:
We don't need `isLoaded` now because I've rebased with master. Can you update this so that it's not used? We don't need the connect at all.